deadbelly/breathEZ
# Feature/select tests

### What does this PR do? 
>Clearly and concisely describe the feature. 
- sets up `fixtures` as stub data
- sets up `integration` files for the main and faq view
- sets up tests for the drop-downs on the main view
### Is this a feature or Fix/Refactor?  
- feature
### Problems Encountered & Resulting Solutions  
>State any problems encountered 
The hangup was attempting to integrate other branches before this branch was ready. The merge led some conflicts in `package-lock.json` and the app could not display. This may have been me not handling the merge conflict in `package-lock.json` properly. 

I did manage to `git revert -m` the commit that merged that version to my working branch and continue on with the tests. I was anticipating that we would have some merge conflicts once pushing up this branch, but right now GitHub is saying that this is a clean merge.
>Describe your code changes in detail for reviewers.
- added `cypress/fixtures`
  - stub data files here
- added `cypress/integration`
  - test files for each view exist here
  - note: nothing in `faq` view tests yet
### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
- `cypress/fixtures`
- `cypress/integration`
### How should this be tested?
- pull down branch (in test branch, not to main)
- run `npm i` to install cypress locally
- run `npm run cypress` to run cypress
- have cypress run the `main-view-spec.js` tests
- observe tests passing with stubbed data
  - indicated by shortened lists for drop-downs
### Steps To Take
>Any questions and possible steps moving forward.
- this branch is mostly set up
- more tests will be added in the near future
### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
No related issues
